File: sprint8/go/lesson2code1.go
func compare(first, second string) bool {
    if len(first) != len(second) {
        return false
    }

    length := len(first)
    for i := 0; i < length; i++ {
        if first[i] != second[i] {
            return false
        }
    }

    return true
}

File: sprint8/C#/lesson2code1.cs
bool Compare(string first, string second) {
    if (first.Length != second.Length) {
        return false;
    }

    int length = first.Length;
    for (int i = 0; i < length; ++i) {
        if (first[i] != second[i]) {
            return false;
        }
    }

    return true;
}
